src/Formik.tsx

```diff
@@ -257,7 +257,7 @@ export function useFormik<Values = object>({
         props.validate ? runValidateHandler(values) : {},
       ]).then(([fieldErrors, schemaErrors, validateErrors]) => {
         const combinedErrors = deepmerge.all<FormikErrors<Values>>(
-          [fieldErrors, schemaErrors, validateErrors],
+          [fieldErrors, schemaErrors, validateErrors] as FormikErrors<Values>[],
```

Comment:
`validateSchemaErrors` should return a `Promise<FormikErrors<Values>>`, then this cast can be removed.

Reply:
I thought I tried that and TS was yelling. Let me do it again